dl/pom.xml

```diff
@@ -239,4 +240,27 @@
             </plugin>
         </plugins>
     </reporting>
+    
+    <profiles>
+    	<profile>
+    		<id>parallel-tests</id>
+   			<properties>
+      			<runSpecsInParallel>true</runSpecsInParallel>
+      			<tagsToExclude>com.intel.analytics.bigdl.tags.Serial</tagsToExclude>
+   			</properties>
+		</profile>
+		<profile>
+    		<id>serial-tests</id>
+   			<properties>
+      			<runSpecsInParallel>false</runSpecsInParallel>
+      			<tagsToExclude>com.intel.analytics.bigdl.tags.Parallel</tagsToExclude>
+   			</properties>
+		</profile>
+		<profile>
+    		<id>serial-all</id>
+   			<properties>
+      			<runSpecsInParallel>false</runSpecsInParallel>
+   			</properties>
+		</profile>
+    </profiles>
```

Comment:
Did you test the maven command `mvn test -Pparallel-tests,serial-tests` to make it works for our case? Because if the PR is merged without the test, jenkins may not work to check pr, so maybe it is better to check and test it before merge, thanks a lot. :-)